JustSaying/IMessageResponseLogger.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Threading.Tasks;
+using JustSaying.AwsTools.MessageHandling;
+using JustSaying.Models;
+
+namespace JustSaying
+{
+    public interface IMessageResponseLogger
+    {
+        /// <summary>
+        /// Capture response of message after it has been sent.
+        /// </summary>
+        Action<MessageResponse, Message> ResponseLogger { get; set; }
```

Comment:
It would be simpler to just expose these as normal method signatures rather than properties that redirect to an `Action` and a `Func`.

Reply:
I agree with Shayne. If it's properties that you have to set, it's simpler to have properties on a DTO `public interface MessageResponseLoggerConfig`. If it's an interface that you have to implement, then methods.